doc/source/license.rst

```diff
@@ -7,6 +7,11 @@ License
 
 OPEN SOURCE LICENSE AGREEMENT
 =============================
+Creative Commons license (CC BY 4.0)
```

Comment:
please see my comment.  I think this needs to be the exact form of the license from LICENSE.rst.

Reply:
OK -- I looked more carefully and it's CC0 license (meaning no copyright?)